tap_universal_file/client.py

```diff
@@ -17,6 +18,41 @@
     from singer_sdk.tap_base import Tap
 
 
+def patched_is_boolean_type(property_schema: dict) -> bool | None:
```

Comment:
@edgarrmondragon  do we want to just get this into the SDK? 

We did something similar here in tap-postgres

https://github.com/MeltanoLabs/tap-postgres/blob/47b61b6c018c8272cec23f39244c5961bb730b72/tap_postgres/client.py#L34-L54

Reply:
Yeah, I think we need to fix this upstream in the SDK.

Reply:
I'll create an issue for it.

Reply:
https://github.com/meltano/sdk/issues/2553